### DISCUSSION TITLE
[课程]预言机 直达第三阶段

### DISCUSSION THREAD

**Discussion by uvd** [hoh-zone/SuiStartrek#117] [Materials]
视频+文章或者ppt
1. 什么事预言机
2. 预言机可以做什么
3. sui天气预言机介绍和实战
4. [Pyth](https://pyth.network/) 介绍和实战
5. [SupraOracles](https://supraoracles.com/) 介绍和实战
6. [Mysten Oracles](https://mystenlabs.com/blog/simple-and-meta-oracles-on-sui) 介绍和实战

**Comment by philo-veritas** [comment]
认领
